libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.TextInput.schema

```diff
@@ -3,7 +3,7 @@
     "$role": "unionType(Microsoft.IDialog)",
     "type": "object",
     "title": "TextInput Dialog",
-    "description": "This represents a dialog which gathers a text from the user",
+    "description": "Collection information - Ask for a word or sentence.",
```

Comment:
>ollection informatio [](start = 21, length = 20)

feels really awkward

Reply:
Sure. Keeping it as is for now for lack of better options. Let's revisit later if we have an alternate.